### ISSUE TITLE
Config: Support `MARIADB_AUTO_UPGRADE` and `MARIADB_*` prefix for environment variables

### ISSUE THREAD

**Issue by lastzero** [yobasystems/alpine-mariadb#60]
When I tested your image for our @photoprism config examples at https://dl.photoprism.app/docker/, I noticed that it does not seem to support the same env variables as the original image (which is not available for ARMv7 though):

https://hub.docker.com/_/mariadb

Adding `MARIADB_AUTO_UPGRADE` would be very useful as well, so inexperienced users don't need to run migrations manually:

```yaml
MARIADB_AUTO_UPGRADE: "1"
MARIADB_DATABASE: "photoprism"
MARIADB_USER: "photoprism"
MARIADB_PASSWORD: "insecure"
MARIADB_ROOT_PASSWORD: "insecure"
```

Unfortunately, we are very busy with our own project, so I don't have time to contribute/send a PR. If anyone has time to implement this, I think it would be very useful for others. Thank you very much!

**Comment by dominictayloruk** [comment]
I haven't realised that the official MariaDB had changed the MYSQL_* to MARIADB_* variables, i can look into that.

Auto upgrade can be run by `mysql_upgrade` within the container. This could be done within the scripts after container start.

**Comment by dominictayloruk** [comment]
This should be fixed with [e206b79](https://github.com/yobasystems/alpine-mariadb/commit/e206b79615df6dd61184bb6ab374e4763f16e38b)

`yobasystems/alpine-mariadb:10.11.8-11`

**Comment by dominictayloruk** [comment]
Auto update just needs adding as a script. I'll create a separate issue for that.
